test/Api.IntegrationTest/SecretsManager/Controllers/ServiceAccountsControllerTests.cs

```diff
@@ -943,6 +929,22 @@ private async Task<(Guid OrganizationId, List<Guid> ServiceAccountIds)> SetupLis
         return (org.Id, serviceAccountIds);
     }
 
+    private static ApiKey CreateTestApiKey(Guid serviceAccountId, string name = "Test Name", DateTime? expiresAt = null)
+    {
+        return new ApiKey
+        {
+            ServiceAccountId = serviceAccountId,
+            Name = name,
+            ExpireAt = expiresAt ?? DateTime.MinValue,
```

Comment:
```suggestion
            ExpireAt = expiresAt,
```

Setting `ExpireAt` to null is valid and is how we set non expiring API keys. 

Having `DateTime.MinValue` would be creating the ApiKey where it is already expired, which I don't think we would want that as the default for tests.

Reply:
Didn't even look to see that it was nullable. Good catch, fixed!